src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -143,8 +187,8 @@ ucs_status_t uct_tcp_sockcm_ep_send_priv_data(uct_tcp_sockcm_ep_t *cep)
 
 static ucs_status_t uct_tcp_sockcm_ep_server_invoke_conn_req_cb(uct_tcp_sockcm_ep_t *cep)
 {
-    uct_tcp_sockcm_priv_data_hdr_t      *hdr = (uct_tcp_sockcm_priv_data_hdr_t *)
-                                               cep->comm_ctx.buf;
+    uct_tcp_sockcm_priv_data_hdr_t      *hdr            = (uct_tcp_sockcm_priv_data_hdr_t *)
```

Comment:
maybe we don't need the extra whitespaces between the type and the name of the variable like in other cases, wdyt?